app/src/main/java/com/kickstarter/ui/activities/CommentsActivity.kt

```diff
@@ -58,8 +59,18 @@ class CommentsActivity :
             .observeOn(AndroidSchedulers.mainThread())
             .subscribe(this::setEmptyState)
 
+        viewModel.outputs.insertComment()
+            .compose(bindToLifecycle())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe {
+                adapter.insertData(it, 0)
+                binding.commentsRecyclerView.scrollToPosition(0)
+            }
+
         binding.commentComposer.setCommentComposerActionClickListener(object : OnCommentComposerViewClickedListener {
             override fun onClickActionListener(string: String) {
+                viewModel.inputs.postComment(string, DateTime.now())
```

Comment:
could you extract this two lines into a private method? something like `private fun postComment(comment)`

Reply:
ok sure